custom_dict.txt

```diff
@@ -118,12 +120,13 @@ DockerHub
 etcd
 filesystem
 Finchley
-Github
+[G,g]ithub
```

Comment:
Shouldn't it always be GitHub (capital G, capital H)? (I believe URLs should be getting ignored already...)

Reply:
I'll fix that.